src/helpers/reporthelper.js

```diff
@@ -169,7 +169,7 @@ const reporthelper = function () {
             wSummaries[1] = null;
             wSummaries[2] = { ...wS[0] };
           }
-        } else { // When two entries.
+        } else if (wS.length === 2) { // When two entries.
```

Comment:
I see that the code change has been based on the previous comment "When two entries". But the actual file's logic works for 2 as well as 3 since the indexing starts from 0. Since there are critical bugs going, i suggest leaving this as original and be merged in later version.